proposals/gc/MVP.md

```diff
@@ -637,8 +637,6 @@ where:
   - `(ref null1? ht1)\(ref null ht2) = (ref ht1)`
   - `(ref null1? ht1)\(ref ht2)      = (ref null1? ht1)`
 
-Note: Cast instructions do _not_ require the operand's source type to be a supertype of the target type. It can also be a "sibling" in the same hierarchy, i.e., they only need to have a common supertype (in practice, it is sufficient to test that both types share the same top heap type.). Allowing so is necessary to maintain subtype substitutability, i.e., the ability to maintain well-typedness when operands are replaced by subtypes.
-
```

Comment:
Why is this no longer necessary / true / helpful?

Reply:
It was primarily meant as an explanation why the earlier rules had this seemingly weird side condition about a common supertype, instead of requiring that the (explicit) target type has to be a subtype of the (implicit) source type. Since the source type is now explicit as well, and the "real" operand type no longer occurs in the rule itself (but can be weakened separately via subsumption), I though that the explanation became both confusing and unnecessary.

Reply:
Fair enough. I think a call-out like this is probably still helpful since folks might not be thinking about subsumption when trying to understand the rules, but of course we cannot add an unbounded number of such helpful notes.